File: Computational Mechanics/2 (C++)/2.1 (Getting Started)/2.1.4 (Conditional Statements)/nestedif_alt.cpp
#include <iostream>

using namespace std;

int main() {

	string my_state = "California";
	
	string my_county = "Riverside County";
	
	if (my_state == "California" and my_county == "Riverside County") {
		cout << "Your county seat is Riverside." << endl;
	}
	else if (my_state == "California" and my_county == "San Bernadino County") {
		cout << "Your county seat is San Bernadino." << endl;
	}
	else if (my_state == "New Mexico" and my_county == "Los Alamos County") {
		cout << "Your county seat is Los Alamos." << endl;
	}
	else if (my_state == "Florida" and my_county == "Miami-Dade County") {
		cout << "Your county seat is Miami." << endl;
	}
	else {
		if (my_state == "California") {
			cout << "I'm only programmed for Riverside and San Bernadino Counties!" << endl;
		}
		else if (my_state == "New Mexico") {
			cout << "I'm only programmed for Los Alamos County!" << endl;
		}
		else if (my_state == "Florida") {
			cout << "I'm only programmed for Miami-Dade County!" << endl;
		}
		else {
			cout << "I'm not programmed to tell you your county seat!" << endl;
		}//end if
	}//end if

	return 0;

}//end main

File: Computational Mechanics/3 (Python)/2.1 (Getting Started)/2.1.4 (Conditional Statements)/nestedif_alt.py
my_state = "California"
my_county = "Riverside County"

if (my_state == "California" and my_county == "Riverside County"):
	print("Your county seat is Riverside.")
elif (my_state == "California" and my_county == "San Bernadino County"):
	print("Your county seat is San Bernadino.")
elif (my_state == "New Mexico" and my_county == "Los Alamos County"):
	print("Your county seat is Los Alamos.")
elif (my_state == "Florida" and my_county == "Miami-Dade County"):
	print("Your county seat is Miami.")
else:
	if my_state == "California":
		print("I'm only programmed for Riverside and San Bernadino Counties!")
	elif my_state == "New Mexico":
		print("I'm only programmed for Los Alamos County!")
	elif my_state == "Florida":
		print("I'm only programmed for Miami-Dade County!")
	else:
		print("I'm not programmed to tell you your county seat!")
	#end if
#end if
